test.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Can we rename this to `.docker-compose-config.json`? That way it doesn't show up in ls output.

~Also would like to add this to the `.dockerignore` file~ NVM that wouldn't work since it's needed for compose. It's probably fine.

Reply:
Done!